### ISSUE TITLE
Add css class on aria-describedby elements to hide PII from Datadog

### ISSUE THREAD

**Issue by powellkerry** [department-of-veterans-affairs/va.gov-team#64871]
There is a request to add the `dd-privacy-hidden` class to aria-describedby elements. This class will hide the content of these elements from Datadog. The concern is that these elements could contain PII/PHI and should be hidden from Datadog analytics.

See support request: https://dsva.slack.com/archives/CBU0KDSB1/p1693922434696939

### Acceptance criteria

The class `dd-privacy-hidden` class is added to all aria-describedby elements 

### Point(s) of contact

@Mottie - Robin Garrison

### What type of issue is this? 
- [ ] Bug
- [ ] Request new feature
- [x ] Request update to existing feature
- [ ] Documentation change
- [ ] Other (please describe)

### How to configure this issue
- [ ] When ticket is opened, assign to Epic [#12765 Forms library user feedback and requests to be triaged](https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/12765)
- [ ] FE Tools team will triage and add to [Design System Governance Process Project Board](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/projects/3) as applicable

**Comment by Mottie** [comment]
PR opened

**Comment by micahchiang** [comment]
https://github.com/department-of-veterans-affairs/component-library/pull/833
